spring/src/test/java/org/axonframework/spring/config/annotation/SpringBeanParameterResolverFactoryTest.java

```diff
@@ -27,6 +27,7 @@
 import org.axonframework.spring.config.AnnotationDriven;
 import org.junit.jupiter.api.*;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.NoUniqueBeanDefinitionException;
```

Comment:
It would be good to note in the class level JavaDoc of this test class, that both the `SpringBeanParameterResolverFactory` as well as the `SpringBeanDependencyResolverFactory` are tested through this class.
Maybe even change the class name to be a bit clearer there too.